public/media_engagement/imaqal_media_engagement.js

```diff
@@ -223,9 +223,11 @@ const update = (data) => {
     // Create line path element for failed line graph
     const total_failed_path = total_failed_sms_graph.append('path');
 
-    let color = d3.scaleOrdinal(d3.schemeCategory10);
-    let colorReceived = d3.scaleOrdinal(d3.schemeCategory10).domain(receivedKeys)
-    let colorSent = d3.scaleOrdinal(d3.schemeCategory10).domain(sentKeys)
+    // custom color scheme
+    color_scheme = ["#e6194B", "#f58231", "#3cb44b", "#9A6324", "#4363d8", "#800000", "#f032e6", "#911eb4"]
```

Comment:
Good job @DanielMwendwa. Telegram & Telesom, Somtel & Natiolink are still closely similar please look for a different color for any one of the MNOs and share the colors in Tech & Data channel.

Reply:
Closed: in #83 since I'm still working on the relative path for the navigation links on this branch.